MBBSEmu.Tests/ExportedModules/Majorbbs/memset_Tests.cs

```diff
@@ -34,8 +35,15 @@ public void memset_Test(byte[] bufMem, ushort valueToFill, ushort memSize, byte[
                 });
 
             //Verify Results
+            var expected = new byte[memSize];
+
+            for (var i = 0; i < memSize; i++)
+                expected[i] = (byte) valueToFill;
+            
             var dstArray = mbbsEmuMemoryCore.GetArray("SETMEMORY", memSize);
 
+            Assert.Equal(0, mbbsEmuCpuRegisters.AX);
```

Comment:
memset should return the first argument as well (`dest`)

Reply:
memset is void as well, so I removed the asserts
```
        ///     Sets the first num bytes of the block of memory with the specified value
        ///
        ///     Signature: void memset(void *ptr, int value, size_t num);
```

Reply:
This comment is wrong inside our code. `string.h` has the actual definition, and it's `void _FAR * _RTLENTRYF _EXPFUNC memset(void _FAR *__s, int __c, size_t __n);`

**void _FAR * **

So it actually does return a pointer value, and mbbsemu is wrong

Reply:
Fixed.